### PR TITLE
Avoid warning logs on USB device unplugging

### PR DESCRIPTION
**Pull request by emaxx-google**
Downgrade a bunch of logs to the INFO level, so that when a smart card
reader is unplugged there are no warnings/errors appearing in the
application log. This is mainly to avoid confusing users who see a red
"Errors" button in chrome://extensions whenever we log a warning/error
and might interpret it as a real problem (which in this case it's not).

### CODE REVIEW

File: smart_card_connector_app/src/manifest.json.template [hunk 1]
```diff
@@ -1,7 +1,5 @@
 {
-${if CONFIG=Debug}
```
Comment (Fabian-Sommer) [inline smart_card_connector_app/src/manifest.json.template:2]: Is this change intentional?

**Review by Fabian-Sommer — APPROVED** [review]
LGTM

File: smart_card_connector_app/src/manifest.json.template (hunk 1 above)
Comment (emaxx-google) [reply smart_card_connector_app/src/manifest.json.template]: Oops. Thanks.
